rcl/test/rcl/test_graph.cpp

```diff
@@ -101,6 +104,12 @@ class CLASSNAME (TestGraphFixture, RMW_IMPLEMENTATION) : public ::testing::Test
     ret = rcl_wait_set_init(
       this->wait_set_ptr, 0, 1, 0, 0, 0, 0, this->context_ptr, rcl_get_default_allocator());
     ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
+
+    rmw_topic_info_array_t valid_topic_info_array = {
```

Comment:
Any reason you didn't do:
```c
valid_topic_info_array = {0};
```
since that will init all values in the struct to 0.

Reply:
I learnt that this was possible from your previous review. I had implemented this before that review and kinda missed this one, gah! Thanks for this though!